### PR TITLE
Add notebook classic theme,

### PR DESCRIPTION
**Pull request by jdfreder**
also don't outline rendered markdown on focus
and fix output prompt padding

classic-theme

### CODE REVIEW

**Comment by rgbkrk** [comment]
omg, so classic

File: src/notebook/index.js [hunk 1]
```diff
@@ -111,8 +111,8 @@ ipc.on('main:load', (e, launchData) => {
                 />
             }
             <NotificationSystem ref="notificationSystem" />
-            <link rel="stylesheet" href={`../static/styles/theme-${this.state.theme}.css`} />
             <link rel="stylesheet" href="../static/styles/main.css" />
+            <link rel="stylesheet" href={`../static/styles/theme-${this.state.theme}.css`} />
```
Comment (rgbkrk) [inline src/notebook/index.js:115]: What's the reason for flipping these? I'm assuming that the CSS vars aren't defined at first, then Chrome resolves it?

File: src/main/menu.js [hunk 2]
```diff
@@ -192,6 +192,10 @@ export const view = {
           label: 'Dark',
           click: createSender('menu:theme', 'dark'),
         },
+        {
+          label: 'Classic',
+          click: createSender('menu:theme', 'classic'),
+        },
```
Comment (rgbkrk) [inline src/main/menu.js:198]: We're charting into that "should we generate this" territory. :wink:

File: src/notebook/index.js (hunk 1 above)
Comment (jdfreder) [inline src/notebook/index.js:115]: User CSS should always be loaded after all base CSS been loaded.  This increases the specificity of the user's CSS selectors.  IOW it makes it so user CSS doesn't have to use `!important`.  Additionally, it allows CSS vars that are declared in the base to be overridden in the theme files, which is not possible without this change.

Comment (rgbkrk) [inline src/notebook/index.js:115]: 👍

**Pull request merged**